js_modules/dagster-ui/packages/ui-components/.storybook/preview.js

```diff
@@ -85,6 +85,8 @@ export const decorators = [
     themes: {
       light: 'themeLight',
       dark: 'themeDark',
+      lightHighContrast: 'themeLightHighContrast',
+      darkHighContrast: 'themeDarkHighContrast',
```

Comment:
I guess these should be called `themeLightNoRedGreen` or something like that? Not really "high contrast" after all.

Reply:
Updated